config/locales/en.yml

```
@@ -46,11 +46,18 @@ en:
       activity: Sign-in attempts and changes to account details
       description: <p class="govuk-body">Read the <a class="govuk-link" href="https://www.gov.uk/government/publications/govuk-accounts-trial-full-privacy-notice-and-accessibility-statement" data-module="gem-track-click" data-track-category="account-manage" data-track-action="security" data-track-label="privacy-notice">full GOV.UK accounts privacy notice</a> to find out how we collect, process and use information about you.</p>
       event:
-        change_email: Changed email address
-        change_password: Changed password
-        change_phone: Changed phone number
-        login: Signed in
-        report_suspicious_activity: Report suspicious activity
+        account_locked: Password verification failed too many times, account locked for 1 hour.
+        email_change_requested: Change of email address has been requested
+        email_changed: Change of email address has been confirmed.
+        login_failure: Entered an incorrect password when logging in.
+        login_success: Signed in
+        manual_account_unlock: Account unlocked by clicking link.
+        password_changed: Changed password
+        password_reset_request: Requested a password reset link.
+        password_reset_success: Reset password
+        two_step_changed: Changed phone number
+        two_step_verification_failure: Entered an incorrect 2fa code when logging in.
+        two_step_verification_success: Entered the correct 2fa code when logging in.
```

It may be more appropriate to label this `phone_number_changed`?
Also, I'm wondering whether to call these "additional factors" and make a more dynamic event that shows what the additional factor is - albeit, currently it's only SMS - might make it more future proof? 
```suggestion
        phone_number_changed: Changed phone number
        additional_factor_added: Additional factor added
        additional_factor_changed: Additional factor changed
        additional_factor_verification_failure: Additional factor was incorrect when logging in.
        additional_factor_verification_success: Additional factor was correct when logging in.
```